nnunet/training/network_training/nnUNetTrainerV2_DDP.py

```diff
@@ -320,9 +319,6 @@ def run_training(self):
         we also need to make sure deep supervision in the network is enabled for training, thus the wrapper
         :return:
         """
-        if self.local_rank == 0:
```

Comment:
Please leave this in here. Otherwise each worker will save this

Reply:
Done.